conda_smithy/cli.py

```diff
@@ -345,6 +376,31 @@ def __call__(self, args):
         else:
             print("Drone registration disabled.")
 
+        if args.cirun:
+            print("Cirun Registration")
+            if args.remove:
+                if args.cirun_resources:
+                    to_remove = args.cirun_resources
+                else:
+                    to_remove = ["*"]
+
+                print(f"Cirun Registration: resources to remove: {to_remove}")
+                for resource in to_remove:
+                    conda_smithy.cirun_utils.remove_repo_from_cirun_resource(
+                        owner, repo, resource
+                    )
+            else:
+                print(
+                    f"Cirun Registration: resources to add to: {owner}/{repo}"
+                )
+                conda_smithy.cirun_utils.enable_cirun_for_project(owner, repo)
```

Comment:
```suggestion
                response = conda_smithy.cirun_utils.enable_cirun_for_project(owner, repo)
                response.raise_for_status()
```

We do not want to continue on errors, do we? Maybe we should even do that on the utility functions.

Otherwise, with a bad token, we get:

```
Cirun Registration
Cirun Registration: resources to add to: conda-forge/cf-autotick-bot-test-package-feedstock
Enabling cirun for conda-forge/cf-autotick-bot-test-package-feedstock ...
──────────────────────────────────────────────────────  ──────────────────────────────────────────────────────
Error
{
  "message": "Bad credentials",
  "documentation_url": "https://docs.github.com/rest"
}
[18:50:03] Response Status Code: 401                                                               utils.py:52
──────────────────────────────────────────────────────  ──────────────────────────────────────────────────────
Heroku webservice registration disabled.

CI services have been enabled. You may wish to regenerate the feedstock.
Any changes will need commiting to the repo.
```

But the request actually failed.

Reply:
The cirun client will check for errors now.